### ISSUE TITLE
Gifox immediately aborts recording after hitting record

### ISSUE THREAD

**Issue by samism** [Gifox/Tracker#241]
Thank you for this wonderful app.

I recently updated to 2.4.0 and regardless of how I attempt to record, the recording immediately cancels. I tried turning the permissions on and off, restarting, and re-installing. No luck. I use Gifox quite a lot for my job, so I was wondering if this was a known issue.

Sam

**Comment by iby** [comment]
> Thank you for this wonderful app.

That aborts recording… 😅 But really thank you for the compliments and positive attitude, always so wonderful to hear!

The latest update comes with a gigantic rework to address some very specific issues that a distinct group of users had. We had another report today and looking into this. Could you please send a message from within an app to include diagnostics info. Very likely to have more some questions, would be great if you could feedback.

![image](https://user-images.githubusercontent.com/1086845/172690682-c6f6801a-0b71-484d-a82c-2ea190370d2e.png)

**Comment by toptaljj** [comment]
I sent a message to support with diagnostic info.  Here is a snippet:
```
2022-06-08 21:52:20.546 [EVENT] Started making new selection.
2022-06-08 21:52:23.032 [EVENT] Finished making new selection: ScreenSelection(mode: area, frame: <x:55 y:928 w:1418 h:836>, screen: <main, x:0 y:960 w:1536 h:960>, scale: 2.0, config: none).
2022-06-08 21:52:24.392 [EVENT] Pressed start recording button in screen selector toolset view.
2022-06-08 21:52:24.515 [INFO] Screen recording session changed state: selecting → recording.
2022-06-08 21:52:25.671 [ERROR] ScreenCaptor was interrupted with error: nil
2022-06-08 21:52:25.675 [ERROR] Screen recording session has failed with error: unexpectedRecorderState(expected: Media.Recorder.State.recording, actual: Media.Recorder.State.finishing).
```

**Comment by alexander-dominguez** [comment]
👆 My logs look same as above post and I too am experiencing this issue.

**Comment by iby** [comment]
Friends, thank you everyone for reporting. We have a fix, will be releasing an update shortly (today or tomorrow)! ✌️

**Comment by iby** [comment]
This should be fixed in 2.4.1 – already available in the direct-download and Setapp versions, will be released in Mac App Store shortly once reviewed. Thanks again for flagging this up, please us let know there's any more weirdness with recording or selecting! Thanks everyone for reporting and being patient! 🙏
